### DISCUSSION TITLE
OpenWrt 22.03.0 - Stable Release

### DISCUSSION THREAD

**Discussion by tommiefr** [nicholas-opensource/OpenWrt-Autobuild#32] [General]
Any update news about this version?
Will this one be released?

**Comment by tommiefr** [comment]
Any news about this?

**Reply by nicholas-opensource** [reply]
v22.03.0 will be released in few days, and currently testing the backport mglru by Google

**Reply by tommiefr** [reply]
Great!!

**Reply by nicholas-opensource** [reply]
Don't expect too much from the new version v22.03.0, openwrt has released a semi-finished version, and there are still many bugs, especially everything related to firewall4.And dnsmasq stable version v2.87 has not been released yet, all went wrong now...

**Reply by tommiefr** [reply]
Ok, than i stay at OpenWrt 21.02.1. this one is going strong for over 180 days now!

**Comment by tommiefr** [comment]
Just saw your new build and the new one released already. whats your advice, is this version stable enough for daily use?

Or will you update to:

https://forum.openwrt.org/t/openwrt-22-03-2-second-service-release/139951

**Reply by nicholas-opensource** [reply]
stable enough, v22.03.2 will release today

**Comment by tommiefr** [comment]
Any news about the 22.03.2 version? I have all setup and ready to install

**Reply by nicholas-opensource** [reply]
May be released in a few days due to some upstream issues

**Reply by nicholas-opensource** [reply]
There is basically not much difference between .1 and .2 if don't use OpenWrt  wireless

**Reply by tommiefr** [reply]
Great! Thanks for your quick reply! Hope you can solve the issues!

**Reply by tommiefr** [reply]
> There is basically not much difference between .1 and .2 if don't use OpenWrt wireless

True, think i'll just install the latest version as i use the r2s.
Thinking about upgrading to r5s. Will you release the version of this device as well?

**Reply by nicholas-opensource** [reply]
No such plans. First of all openwrt does not support this device and rk3568 does not provide enough source code to support it very well. The R5S won't bring any more performance than the R2S, and a terrible 2.5GbE NIC
